procrustes/permutation.py

```diff
@@ -136,12 +136,24 @@ def permutation(
     return ProcrustesResult(new_a=new_a, new_b=new_b, t=array_u, error=error)
 
 
-def permutation_2sided(array_a, array_b, transform_mode="single",
-                       remove_zero_col=True, remove_zero_row=True,
-                       pad_mode="row-col", translate=False, scale=False,
-                       mode="normal1", check_finite=True, iteration=500,
-                       add_noise=False, tol=1.0e-8, kopt=False, kopt_k=3,
-                       weight=None):
+def permutation_2sided(
+        array_a,
+        array_b,
+        transform_mode="single",
+        pad=False,
+        unpad_col=False,
+        unpad_row=False,
+        translate=False,
+        scale=False,
+        mode="normal1",
+        check_finite=True,
+        iteration=500,
+        add_noise=False,
+        tol=1.0e-8,
+        kopt=False,
+        kopt_k=3,
```

Comment:
I would combine these and just have one argument. In other words, if `kopt=None`, the kop algorithm is skipped, However, if is given an integer value, the kopt step is added. Let me know if you have other suggestions.

Reply:
This is resolved in bf8cc6f